stonesoup/sensormanager/base.py

```diff
@@ -111,7 +111,9 @@ def choose_actions(self, tracks, timestamp, nchoose=1, **kwargs):
             Time at which the actions are carried out until
         nchoose : int
             Number of actions from the set to choose (default is 1)
-
+        return_reward: bool
+            Whether to return the reward for chosen actions (default is False)
+            When True, returns a tuple of 1d arrays: (dictionaries of chosen actions, rewards)
         Returns
         -------
         : dict
```

Comment:
Sorry I'm being picky...

```suggestion
        return_reward: bool
            Whether to return the reward for chosen actions (default is False).
            When True, returns a tuple of 1d arrays: (dictionaries of chosen actions, rewards)
        Returns
        -------
        : list(dict) or (list(dict), :class:`numpy.ndarray`)
```

Then probably also clarify in the line below that the dict is the configuration of ":class:`~.Sensor`:` [:class:`~.Action`]"and the array contains the corresponding reward.